drivers/SmartThings/zigbee-button/src/ewelink/init.lua

```diff
@@ -32,6 +33,11 @@ local function can_handle_ewelink_button(opts, driver, device, ...)
   return false
 end
 
+local function do_configure(driver, device)
+  device:configure()
+  device:send(device_management.build_bind_request(device, OnOff.ID, driver.environment_info.hub_zigbee_eui))
```

Comment:
is bind request for PowerConfiguration sent by the scripting engine by default ?